### PR TITLE
Flake8 update

### PR DESCRIPTION
**Pull request by peasant98**
Changing some of the code based on flake8. Everything is now in the Python application yml file.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": true
```
Comment (watakandai) [inline .vscode/settings.json:2]: Danke

File: robotic_skin/algorithm/__init__.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
 """
 init for robotic skin algorithm
 """
-from .convert_to_lowertriangular_matrix import ConvertToLT
+from .convert_to_lowertriangular_matrix import ConvertToLT  # noqa F401
```
Comment (watakandai) [inline robotic_skin/algorithm/__init__.py:4]: What does ` # noqa F401` mean?

**Review by watakandai — APPROVED** [review]

**Review by watakandai — APPROVED** [review]

**Comment by krishnachaitanya7** [comment]
It's Merge time bro!

**Comment by peasant98** [comment]
finally worked. Let's get it.

**Pull request merged**